### ISSUE TITLE
Wrong loss or Missing "LogSoftmax" in your diagram

### ISSUE THREAD

**Issue by drozzy** [PyTorchLightning/pytorch-lightning#1856]
I think there might be a mistake in the big diagram: https://pytorch-lightning.readthedocs.io/en/latest/new-project.html

Either you:

1) meant to apply log_softmax first, OR
2) meant to apply pytorch's crossentropy loss instead (which takes logits and applies both log_softmax and nll_loss)?

See: https://pytorch.org/docs/stable/nn.functional.html#cross-entropy

![pt_to_pl](https://user-images.githubusercontent.com/140710/82129611-28938f80-9792-11ea-992e-8bf3a6c403d9.jpg)

**Comment by drozzy** [comment]
I mean considering it's the main diagram for the library, I think you should make sure it's correct.

**Comment by ydcjeff** [comment]
@drozzy that diagram has been updated with the new GIF

**Comment by drozzy** [comment]
The new gif just removed the argmax part (which was not needed), but it is still wrong.
 
It did not add `log_softmax` or make use of `crossentropy` loss as per my previous suggestion.

P.S.: It is also much harder to see what's going on with the animations, so I had to take a screen shot.

<img width="407" alt="Screen Shot 2020-07-29 at 3 50 41 PM" src="https://user-images.githubusercontent.com/140710/88846217-4da17700-d1b3-11ea-8a80-369f58b2141d.png">

**Comment by awaelchli** [comment]
@drozzy The docs now have videos you can pause.
But the problem with the example remains. I agree this should really be changed.
For example, one could apply the log_softmax in the model like in this example:
https://github.com/pytorch/examples/blob/master/mnist/main.py

**Comment by williamFalcon** [comment]
oh shoot... yeah, i can update. that code had typos so we fixed to the new one:

https://pytorch-lightning.readthedocs.io/en/latest/callbacks.html

but forgot to update the quick start and intro. thanks for bringing this up!

**Comment by awaelchli** [comment]
Thanks @williamFalcon The animations now all have the correct loss cross entropy. 
There is a new minor issue, the refactored code still has the "pytorch_model" as the forward pass instead.
